### ISSUE TITLE
Setting for global text color

### ISSUE THREAD

**Issue by ux-engineer** [webcat12345/ngx-ui-switch#315]
It seems there is no setting for the text color. While the background, border etc colors are configurable, text color should be also.

**Comment by envision** [comment]
Ended up customizing colors in  _src/app/styles.scss_ like so. As I needed to customize active state border-color, needed to use `!important` because the component's styles are inlined.

```
ui-switch {
  color: white;
  .switch.checked {
    border-color: #007c00 !important;
  }
}
```

**Comment by gregoryalary** [comment]
I just opened a PR that would add this feature to the component through two properties : `checkedTextColor`/`uncheckedTextColor`.

**Comment by cmckni3** [comment]
Will publish to npm when I have a chance this evening

**Comment by gregoryalary** [comment]
This issue should be closed now that 8.3.0 has been published to npm.
